pxr/imaging/hio/OpenEXR/deflate/lib/deflate_decompress.c

```diff
@@ -1081,9 +1081,9 @@ typedef enum libdeflate_result (*decompress_func_t)
 
 #ifdef arch_select_decompress_func
 static enum libdeflate_result
-dispatch_decomp(struct libdeflate_decompressor *d,
-		const void *in, size_t in_nbytes,
-		void *out, size_t out_nbytes_avail,
+dispatch_decomp(struct libdeflate_decompressor * restrict d,
```

Comment:
corresponds to upstream fix at https://github.com/ebiggers/libdeflate/commit/343ec0a6d67767c8ce10595555e8dae0ceb973ef